plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteJdbcConfig.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+import org.apache.ignite.internal.jdbc.thin.ConnectionPropertiesImpl;
+
+import javax.validation.constraints.AssertTrue;
+
+import java.sql.SQLException;
+
+public class IgniteJdbcConfig
+        extends BaseJdbcConfig
+{
+    @AssertTrue(message = "Invalid JDBC URL for Ignite connector")
+    public boolean isUrlValid()
+    {
+        try {
+            new ConnectionPropertiesImpl().setUrl(getConnectionUrl());
+            return true;
+        }
+        catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
instead can we return false.

Reply:
Yes, can. 
But I see the `MySqlJdbcConfig` also throws an exception here, wonder shall we update that implementation as well?

Reply:
Updated